engine/sync_manager.go

```diff
@@ -451,10 +451,7 @@ func (m *SyncManager) worker() {
 	}
 	defer cleanup()
 
-	interval := greatestCommonDivisor(m.config.TimeoutWebsocket, m.config.TimeoutREST)
-	if interval > time.Second {
-		interval = time.Second
-	}
+	interval := min(greatestCommonDivisor(m.config.TimeoutWebsocket, m.config.TimeoutREST), time.Second)
```

Comment:
🐒 Nitpick: time.Second should be a const
```suggestion
	interval := min(greatestCommonDivisor(m.config.TimeoutWebsocket, m.config.TimeoutREST), minSyncInterval)
```